web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
```

Comment:
I agree – the "status" page is about _status of scraping_ more than _the loaded configuration_. The latter is also less frequently needed, so I'd say: keep the top-level "status" page. make it contain a link to config (at `/status/config`), and show the target status directly on `/status`. This stays true to config being a part of status, but a less frequently visited one.